### ISSUE TITLE
Suspicious login attempt in Instagram

### ISSUE THREAD

**Issue by sfatfarma** [LookHin/instagram-photo-video-upload-api#11]
This is still working from my localhost machine, but when I try using it on a server (or on any server I tried so far), a 'Suspicious login attempt' window pops for me in my Instagram account each time when I try to run this script (regardless that I click 'It was me' when prompted in Instagram).

This is happening since 1st November 2018, when Instagram seems to have made some major changes to it's security settings.

I tried also proxies (implemented them myself to this library), but haven't found one that works yet.

**Comment by xosad** [comment]
Yes, because you are probably using the account in a server which is in location A, and you are verifying/unlocking your account in location B. Because of these 2 locations, you are getting that. Use it in the same network/location
